src/mqueryfront/src/query/QueryMatches.js

```diff
@@ -93,8 +93,8 @@ const QueryMatches = (props) => {
                                 />
                             </span>
                             {filters.length > 0 && (
-                                <span class="border rounded p-1 pull-right">
-                                    <FilterIcon color="grey" />
+                                <span class="border rounded p-1 pull-right text-secondary">
+                                    <FilterIcon color="text-secondary" />
```

Comment:
The color isn't necessary if you used `text-secondary` on the span.
```suggestion
                                    <FilterIcon />
```

Reply:
(and it wouldn't work not work, because `text-secondary` is a class and not color)

Reply:
Interesting, but the color (text-secondary) for an icon is set only when it is defined in the class and icon argument. I just leave the grey icon color instead.